src/urllib3/connection.py

```diff
@@ -248,6 +248,12 @@ def connect(self) -> None:
         # not using tunnelling.
         self._has_connected_to_proxy = bool(self.proxy)
 
+        # HTTPConnection, thus no ssl verification
+        # fix 2/4 for https://github.com/urllib3/urllib3/issues/3130
+        # fixes test_proxy_verified_http_http
```

Comment:
Thanks for clarifying what the effect of the change is, but we do not tend to have such comments merged in the code base

Reply:
Some cause-effect documentation helps understanding code, especially during debugging/development.

As a newcomer i can tell you that sources are under-documented.

Can we keep for now, and remove later (if you still insist)?

Reply:
=> removed this comment